web/src/lib/domain-logic/filterCertifications.ts

```diff
@@ -3,9 +3,13 @@ import { Certification } from "@/lib/types/types";
 import { Business } from "@businessnjgovnavigator/shared";
 
 export const filterCertifications = (
-  certifications: Certification[],
-  business: Business
+  certifications: Certification[] | undefined,
```

Comment:
[sand] These should probably be designated as optionals (`certifications?`, for example) if they are `| undefined` types.